src/hubAuth.ts

```diff
@@ -73,13 +69,21 @@ export const testkitHubAuth = (homeDir: string): void => {
     const jwtKey = path.join(homeDir, 'jwtKey');
     fs.writeFileSync(jwtKey, formatJwtKey());
 
-    shell.exec(
+    const results = shell.exec(
```

Comment:
There is a [bug in shelljs](https://github.com/shelljs/shelljs/issues/981) that ignore `fatal: true` when not set globally, so for now capture results and throw if the command fails.

Reply:
good catch.